run/CESM/geoschem_config.yml

```diff
@@ -32,9 +32,33 @@ operations:
       activate: true
       use_static_bnd_cond: true
     gamma_HO2: 0.2
+    autoreduce_solver:
+      activate: false
+      use_target_threshold:
+        activate: true
+        oh_tuning_factor: 0.00005
+        no2_tuning_factor: 0.0001
+      use_absolute_threshold:
+        scale_by_pressure: true
+        absolute_threshold: 100.0
+      keep_halogens_active: false
+      append_in_internal_timestep: false
 
-  convection:
-    activate: false
+  photolysis:
+    activate: true
+    input_directories:
+      fastjx_input_dir: /see/namelist/file
+      cloudj_input_dir: /see/namelist/file
+    overhead_O3:
+      use_online_O3_from_model: true
+      use_column_O3_from_met: true
+      use_TOMS_SBUV_O3: false
+    photolyze_nitrate_aerosol:
+      activate: true
+      NITs_Jscale_JHNO3: 100.0
+      NIT_Jscale_JHNO2: 100.0
```

Comment:
Heads up, the NIT*scale* tags will change in 14.4.0, see PR #1998.

Reply:
Good to know. I'll change them when I update CESM to use 14.4.